numeral.js

```diff
@@ -394,8 +411,7 @@
             if (format.indexOf('.') === 0) {
                 w = '';
             }
-
-            return ((negP && neg) ? '(' : '') + ((!negP && neg) ? '-' : '') + ((!neg && signed) ? '+' : '') + w + d + ((ord) ? ord : '') + ((abbr) ? abbr : '') + ((bytes) ? bytes : '') + ((negP && neg) ? ')' : '');
+            return ((negP && neg) ? '(' : '') + ((!negP && neg) ? '-' : '') + ((!neg && signed) ? '+' : '') + w + d + ((ord) ? ord : '') + ((abbr && !omitAbbr) ? abbr : '') + ((bytes && !omitAbbr) ? bytes : '') + ((negP && neg) ? ')' : '');
```

Comment:
Maybe we don't want to drift too far from the forked project, but for readability, some line breaks on the appends could really help here.